.eslintrc.yaml

```diff
@@ -103,9 +110,6 @@ rules:
         'ts-nocheck': true,
         'ts-check': false,
     }]
-    '@typescript-eslint/indent': [warn, 4, {
-        SwitchCase: 0
-    }]
```

Comment:
This rule is deprecated, see: https://typescript-eslint.io/rules/indent/

Reply:
The doc says 
```
 Use a separate dedicated formatter instead.
```

So which formatter is used instead?

Reply:
https://github.com/cocos/cocos-engine/pull/15735/files#diff-3894e4104cda190617684d0dc5d55e39e49c63cc6955edf4a446b91ae086bbd2R73